main.tf

```
@@ -8,11 +8,12 @@ locals {
   protection = flatten([
     for config in var.branch_protection : [
       for branch in config.branches : {
-        branch            = branch
-        enforce_admins    = config.enforce_admins
-        push_restrictions = config.push_restrictions
-        required_reviews  = config.required_reviews
-        required_checks   = config.required_checks
+        branch                 = branch
+        enforce_admins         = config.enforce_admins
+        push_restrictions      = config.push_restrictions
+        required_reviews       = config.required_reviews
+        required_checks        = config.required_checks
+        require_signed_commits = config.require_signed_commits
```

Please use correct alphabetical ordering (goes for all locations of course):
```suggestion
        branch                 = branch
        enforce_admins         = config.enforce_admins
        push_restrictions      = config.push_restrictions
        require_signed_commits = config.require_signed_commits
        required_reviews       = config.required_reviews
        required_checks        = config.required_checks
```